add type at rte (#1052)
contao/docs: docs/dev/reference/dca/fields.md

| readonly           | true/false (`bool`)           | Makes the field read only (not supported by all field types).                                                                                                            |
| rgxp               | Regular expression (`string`)    | Limits the user input to certain rules (see paragraph [Regular Expressions](#regular-expressions)).                                                                      |
| rows               | Rows (`integer`)                 | Number of rows (used for `textarea` and `tableWizard` fields).                                                                                                                                   |

> | rte                | Rich text editor file (`string`) | `ace` or `tinyMCE` for example - you can add a type e.g. `ace|js` or `ace|php`. |

Reviewer: ```suggestion
| rte                | Rich text editor file (`string`) | `ace` or `tinyMCE` for example. For `ace` you can also define the type, e.g. `ace|js` or `ace|php`. |
```
